src/components/UTIL_HtmlOutput.component

```diff
@@ -0,0 +1,12 @@
+<apex:component layout="none" id="util_htmloutput">
+    <!-- This component will strip out the whitelisted html tags then encode the rest of the html and finally substitue back in the original tags -->
+    <apex:attribute name="html" type="String" description="The html string to output." required="true" />
+
+    <apex:variable var="temp" value="{!SUBSTITUTE(SUBSTITUTE(SUBSTITUTE(SUBSTITUTE(SUBSTITUTE(html,
```

Comment:
@force2b @davisagli - Super elementary approach to sanitizing the html but we don't use a lot of tags so I wanted to run it past you to see what you think.

Reply:
Eeek I had bigger issues than that in there. I didn't update my variable references either. My links are not properly working after I fixed it. Thanks.